app/serializers/post_serializer.rb

```diff
@@ -298,6 +298,7 @@ def trust_level
 
   def reply_to_user
     {
+      id: object.reply_to_user.id,
       username: object.reply_to_user.username,
```

Comment:
Minor, but you could pass `username` instead of `id` in the `d-editor` component and with that you wouldn't need to add a new attribute to this serializer.

Reply:
I feel that using a username could feel a bit weird–most places pass just the `user_id` for search.

And I have a strong belive that this specific id is not a secret(I was too security concerned)

Reply:
Yeah, no security concerns with the ID. We include it in the basic user serializer. 

Ok to proceed with the `id`.